test/models/job_execution_test.rb

```diff
@@ -125,6 +127,15 @@
     lines.must_include "TAG=v1"
   end
 
+  it 'exports deploy_group information if deploy groups present' do
+    job.update(command: 'env')
+    job.deploy = deploy
+    execute_job
+    lines = job.output.split "\n"
+    lines.must_include "# Deploy URL: #{deploy.url}"
+    lines.must_include "DEPLOY_GROUPS=Pod1 Pod2"
```

Comment:
This is matching against the output of 'env' which won't have the quotes in them.